tft.py

```diff
@@ -532,114 +468,122 @@ def check_if_gold_at_least(num: int) -> bool:
             time.sleep(0.2)
             continue
 
-        if not onscreen(CONSTANTS["game"]["round"]["1-"], 0.9) and not onscreen(CONSTANTS["game"]["round"]["2-"], 0.9):
-            if check_if_gold_at_least(2) and onscreen(CONSTANTS["game"]["gamelogic"]["reroll"]):
-                click_to_middle(CONSTANTS["game"]["gamelogic"]["reroll"])
+        if check_if_gold_at_least(2) and onscreen(CONSTANTS["game"]["gamelogic"]["reroll"]):
+            click_to_middle(CONSTANTS["game"]["gamelogic"]["reroll"])
+            time.sleep(0.2)
+            continue
 
         time.sleep(0.5)
 
-        if check_if_post_game():
-            match_complete()
-            break
-
 
 def end_match() -> None:
     """End of TFT game logic.
 
-    Loops to ensure the various end-of-match scenarios are accounte for to help ensure we make it back to the next find match button.
+    Loops to ensure we are no longer in a game.
 
-    Will dismiss various interruptions and screen such as 'waiting for stats' or 'play again'.
+    Will check for client errors that require a restart.
     """
     counter = 0
-    # added a main loop for the end match function to ensure you make it to the find match button.
-    while not onscreen_multiple_any(find_match_images):
-        bring_league_client_to_forefront()
-        if check_if_client_error() or not league_client_running():
-            return
-        dismiss_interruptions()
-        time.sleep(2)
-        if onscreen_multiple_any(skip_waiting_for_stats_images):
-            logger.info("Skipping waiting for stats")
-            click_to_middle_multiple(skip_waiting_for_stats_images)
-        if onscreen(CONSTANTS["client"]["post_game"]["play_again"]):
-            logger.info("Attempting to play again")
-            bring_league_client_to_forefront()
-            click_to_middle(CONSTANTS["client"]["post_game"]["play_again"], delay=0.5)
-        if onscreen(CONSTANTS["client"]["pre_match"]["quick_play"]):
-            logger.info("Attempting to quick play")
-            click_to_middle(CONSTANTS["client"]["pre_match"]["quick_play"])
-            time.sleep(3)
-        if onscreen(CONSTANTS["client"]["tabs"]["tft"]["subtab_home"]):
-            logger.info("Attempting to select TFT subtab 'home'")
-            click_to_middle(CONSTANTS["client"]["tabs"]["tft"]["subtab_home"])
-            time.sleep(3)
-        if not onscreen_multiple_any(find_match_images) and onscreen_multiple_any(unselected_tft_tabs, precision=0.9):
-            logger.info("Detected that TFT tab is not selected, attempting to select")
-            click_to_middle_multiple(unselected_tft_tabs)
-        # Account for weird scenario of most client UI not loading
-        counter += 1
-        if counter > 60:
+    while True:
+        if counter >= 60:
             restart_league_client()
             return
+        if LCU_INTEGRATION.in_game():
+            counter += 1
+            time.sleep(1)
+            continue

```

Comment:
Does LRU continue to report the user as in-game after the user has already quit? 🤔